ibis/pandas/execution/window.py

```diff
@@ -156,19 +156,31 @@ def trim_with_timecontext(data, timecontext: Optional[TimeContext]):
         context might be adjusted for calculation. Data must be trimmed
         within the original time context before return.
 
+        Params
+        ------
+        data: pd.Series of MultiIndex
```

Comment:
I don't think this is correct. Do you mean pd.Series with MultiIndex?

Reply:
Oh, am I using ``of`` wrong here? 

It looks like this:
```
level_0  time  value
2       2010-01-02 2
3       2010-01-03 3
```
index of this Series is ```MultiIndex(None, Timestamp)```

What's the difference between `of` and `with` here? Or is it just wrong to use `of`?